packages/jokul/package.json

```diff
@@ -486,7 +486,8 @@
         "react-dom": "^18.0.0"
     },
     "peerDependencies": {
-        "react": "^18.0.0"
```

Comment:
Jeg var helt sikker på at jeg allerede hadde flyttet `react` til `dependencies`. Vi importerer masse greier (hooks) direkte fra react så den hører ikke til under peer.

Reply:
Ah, her i autosuggest PRen

https://github.com/fremtind/jokul/pull/4257/files#diff-807b49887b5bf9764896c79feda06c79d8e206a31a5ef68d4272cf36cb6a489d

Hvis du gjør samme endring her så minsker det sjanse for conflicts :)

Reply:
Hvis `react` flyttes til `dependencies` bør typene også flyttes dit, evt. legges under `devDependencies`.

Reply:
Vi bygger jo dette til javascript så @types/react er vel fortsatt ikke nødvendig for at ting skal fungere slik som react er? Det er bare relevant for de som faktisk skal bruke ts-deklarasjonen til noe vil jeg anta.

Reply:
Typene må være en avhengighet for at det skal funke uten automatisk hoisting, slik det er satt opp i monorepo. Derav denne PR-en 😊

Reply:
Jeg tenkte bare at den kun er nødvendig dersom man bruker typescript i konsumenten så jeg var nysgjerrig på hvorfor den måtte ligge i `dependencies` fordi react også gjorde det. Tror ikke det har så mye å si i praksis. Jeg prøvde egentlig bare å varsel om en conflict :)

Reply:
Jeg testet med å bruke .pnpmfile.cjs i https://github.com/Saegrov/mono-jokul-test og det fungerte **ikke** å ha det som `devDependency`. `peerDependency` eller `dependency` løser begge problemet.

Hva som er mest "riktig" vet jeg ikke, men jeg tenker det er greit å ha både `react` og `@types/react` som samme type.

@ivarni  og/eller @piofinn : Bestem hvor det er best dere.

Reply:
Etter som @ivarni la `react` inn i  `dependency` forrige uke så oppdaterte jeg nå med `@types/react` der også

Reply:
For meg er det ett fett, var som sagt bare nysgjerrig på hvorfor :)